src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int COLOR_COUNT = Colors.values().length;
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int colorNumber = random.nextInt(COLOR_COUNT);
+        Colors targetColor = Colors.values()[colorNumber];
+        return targetColor.name();
```

Comment:
[Don’t create temporary variables if you can directly return value from the method](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-temporary-variables-if-you-can-directly-return-value-from-the-method)

Reply:
got it